lib/archethic/p2p/message.ex

```diff
@@ -436,14 +448,24 @@ defmodule ArchEthic.P2P.Message do
     <<250::8, Enum.count(unspent_outputs)::32, unspent_outputs_bin::binary>>
   end
 
-  def encode(%TransactionList{transactions: transactions}) do
+  def encode(%TransactionList{transactions: transactions, more?: false, page: nil}) do
+    transaction_bin =
+      transactions
+      |> Stream.map(&Transaction.serialize/1)
+      |> Enum.to_list()
+      |> :erlang.list_to_bitstring()
+
+    <<251::8, Enum.count(transactions)::32, transaction_bin::bitstring, "FALSE", "PAGE_NIL">>
```

Comment:
I think you can encode in a more efficient way instead of "FALSE", you can use a bitstring: `0::1` , and opposite for "TRUE"